modules/ipc/src/zenoh/dynamic_subscriber.cpp

```diff
@@ -91,12 +88,14 @@ void DynamicSubscriber::onPublisherAdded(const PublisherInfo& info) {
 }
 
 void DynamicSubscriber::onPublisherDropped(const PublisherInfo& info) {
-  throwExceptionIf<InvalidOperationException>(
-      !subscribers_.contains(info.topic),
-      fmt::format("Trying to drop topic {}, but subscriber doesn't exist", info.topic));
+  if (!subscribers_.contains(info.topic)) {
+    LOG(ERROR) << fmt::format("Dropping subscriber for topic: {}, but one doesn't exist", info.topic);
```

Comment:
This description is unclear, what is actually happening?

Reply:
I will rewrite, but basically we are trying to delete a subscriber which doesn't exists in the first place